### ISSUE TITLE
Has the "virtualizer" library to be installed?

### ISSUE THREAD

**Issue by frisso** [netgroup-polito/frog4-virtualizer#7]
It is not clear to me.
If you go in that folder, it seems it has to be installed.
However, the main readme file doesn't report that info.
What should we do?

**Comment by francesco9911** [comment]
Yes, it has to be installed. Now the README file report that info.
commit 53217c91637f5bd37f07b2eb170fd09a509f9dbe in branch new-api
